modules/cells/src/main/java/dmg/cells/nucleus/CellAdapter.java

```diff
@@ -808,7 +808,7 @@ void dumpPinboard()
      */
     @Override
     public void   exceptionArrived(ExceptionEvent ce) {
-        _log.info(" exceptionArrived "+ce);
+        _log.info(" exceptionArrived {}", ce);
```

Comment:
here we probably want to have _log.info(" exceptionArrived {}", ce.toString()); to avoid logging of a stack trace